backend/src/utils/utils.ts

```diff
@@ -16,3 +21,40 @@ const runShellCommand = async (command: string): Promise<string> => {
 
 export const pipe = <T>(...fns: Array<(a: T) => T>) => (x: T): T =>
   fns.reduce((value, func) => func(value), x)
+
+export const getRepositoryFromFilePath = (file: File): string => {
+  return file.name.split('/').slice(0, 2).join('/')
+}
+
+export const getFileNameFromFilePath = (
+  file: File,
+  repositoryName: string
+): string => {
+  return file.name.replace(`${repositoryName}/`, '') || file.name
+}
```

Comment:
what is the purpose of `|| file.name` here?

Reply:
Did not dare to change it because I thought it had some corner case and I haven't created the function. If it's pointless at all times then we can remove.

Reply:
I can't figure when we would return the `file.name` but lets keep it there so we don't break anything